src/agenda/index.js

```diff
@@ -139,6 +146,16 @@ export default class AgendaView extends Component {
     this.state.scrollY.removeAllListeners();
   }
 
+  onHover() {
```

Comment:
These are very specific implementations. If you're adding a feature, first, you should make a separate PR for that and not add it to another issue. Second, you should write an API that lets the user adjust these implementations if required.

Reply:
Sure I'll split the mouse interactions into a second PR. Unless @bkniffler wants it since it is their code :) LMK

For API/adjustable-ness - none of the other interactions have this AFAICT? Am I missing something?

Reply:
I don't think these hover interactions are crucial for web support. In my opinion, they should be moved to a separate PR to help get this one merged sooner.

The right way to do this would probably be to transfer over to React Native's new `Pressable`, and style it like so:

```js
<Pressable
  // react-native-web 0.14 added support for hovered here 🔥
  style={({ hovered, pressed }) => {
    opacity: pressed || hovered ? 0.7 : 1
  }}
/>
```

I'm not sure how focused on backwards-compatibility this library is, though.